source/fuzz/fuzzer_pass_add_vector_shuffle_instructions.cpp

```diff
@@ -60,10 +60,10 @@ void FuzzerPassAddVectorShuffleInstructions::Apply() {
         std::vector<opt::Instruction*> vector_instructions =
             FindAvailableInstructions(
                 function, block, instruction_iterator,
-                [instruction_descriptor](
+                [this, instruction_descriptor](
                     opt::IRContext* ir_context,
                     opt::Instruction* instruction) -> bool {
-                  if (!instruction->type_id()) {
+                  if (!instruction->type_id() || !instruction->result_id()) {
```

Comment:
Elsewhere we always use the order result_id, type_id, so for neatness let's do the same here.

Reply:
Fixed in 35f5730.